examples/config/inventory-osh.ini

```diff
@@ -30,6 +30,3 @@ osh-deploy-admin
 osh-deploy-masters
 osh-deploy-workers
 osh-deployer
-
-[osh-deploy-all-nodes:vars]
```

Comment:
Is this does not work ? or any specific reason/benefit of passing ansible_user in each host definition line?

Reply:
Yes, the variables are evaluated in a group level, not at the host level. Sensu stricto, this var should be at a host level. But I am fine with keeping it. Just bad memories of overrides that had to go back and forth for certain connection plugins :)